server/checkdoc.js

```diff
@@ -0,0 +1,28 @@
+const Doc = require('doc');
+const Taboo = require('taboo');
+
+const inputdoc = (req, res) => {
+  let docinst = new Doc({
+    title: req.body.name,
+    content: req.body.text,
+    owner_id: req.body.owner_id,
+    locked: false,
+    revision_id: 0,
+  });
+
+  let taboo_words_data = Taboo.find({}, function (err, results) {
```

Comment:
This should not exist here. Perhaps create a separate function that retrieves taboo words and returns a list of pairs of indexes. Rename this file to docHandlers?

Reply:
moved into outside function and renamed file